docs/customizing.md

```diff
@@ -180,6 +183,58 @@ Example `pulp_install_plugins`:
          upgrade: true
 ```
 
+### Latest patch release for a minor release of pulpcore
```

Comment:
What I had in mind was:
```suggestion
### Latest patch releases for minor releases of pulpcore and plugins
```

In other words, both pulpcore & plugins stay on the same branch, but get the latest.

In other words, a sysadmin who wants to "patch", but not "upgrade". Assume the "upgrades" are done every 6 months (or when a feature need arises), but they "patch" every month.

In fact, you to make your approach work, extra steps are involved. Users would have to make sure the latest plugin is not too new that it requires newer pulpcore.

The rest of my comments are assuming my suggested approach.